test/quantization/test_quant_primitives.py

```diff
@@ -744,8 +748,8 @@ def test_groupwise_affine_dequantize_tensor_from_qparams(self):
                 zeros = torch.randint(0, 15, (10, 2), dtype=torch.int32)
             if TORCH_VERSION_AT_LEAST_2_5:
                 input_tmp = input
-                if not (
-                    is_device(input.device.type, "cpu") and TORCH_VERSION_AT_LEAST_2_6
+                if (not (check_cpu_version(input.device))) and (
```

Comment:
my comment was actually meant to say that we can encapsulate these format changes to some function `input_tmp = (input[::, ::2] << 4 | input[::, 1::2]).to(torch.uint8)` so we don't need to have these across the codebase since it's pretty error prone and we are not sure when to use them